Regression/WarpX-tests.ini

```diff
@@ -1271,6 +1271,21 @@ tolerance = 1.e-14
 buildDir = .
 inputFile = Examples/Tests/TwoParticle_electrostatic/inputs_3d
 dim = 3
+addToCompileString = DO_ELECTROSTATIC=TRUE
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 1
+numthreads = 2
+compileTest = 0
+doVis = 0
+compareParticles = 0
+tolerance = 1.e-12
+
+[ElectrostaticSphere]
+buildDir = .
+inputFile = Examples/Tests/ElectrostaticSphere/inputs_3d
+dim = 3
```

Comment:
We need to add a line for `analysisRoutine = ` here, that points to the plotting script.